q2_feature_classifier/tests/test_cutter.py

```diff
@@ -18,23 +22,43 @@ class CutterTests(FeatureClassifierTestPluginBase):
 
     def setUp(self):
         super().setUp()
-        self.sequences = Artifact.import_data(
-            'FeatureData[Sequence]', self.get_data_path('dna-sequences.fasta'))
+        seqs = skbio.io.read(self.get_data_path('dna-sequences.fasta'),
+                             format='fasta', constructor=skbio.DNA)
+        tmpseqs = os.path.join(self.temp_dir.name, 'temp-seqs.fasta')
+        skbio.io.write((s for s in islice(seqs, 10)), 'fasta', tmpseqs)
+        self.sequences = Artifact.import_data('FeatureData[Sequence]', tmpseqs)
 
     def test_extract_reads(self):
         # extract_reads should generate a FeatureData(Sequence) full of reads
-        trunc_len = 75
-        trim_left = 5
         f_primer = 'AGAGTTTGATCMTGGCTCAG'
         r_primer = 'GCTGCCTCCCGTAGGAGT'
         extract_reads = feature_classifier.methods.extract_reads
-        result = extract_reads(self.sequences, f_primer=f_primer,
-                               r_primer=r_primer, trunc_len=trunc_len,
-                               trim_left=trim_left, identity=0.9)
-        self.assertEqual(result.reads.type, FeatureData[Sequence])
         inseqs = list(self.sequences.view(DNAIterator))
-        outseqs = list(result.reads.view(DNAIterator))
-        self.assertLessEqual(len(outseqs), len(inseqs))
-        self.assertTrue(bool(outseqs))
-        for seq in outseqs:
-            self.assertEqual(len(seq), trunc_len - trim_left)
+
+        trunc_lens = 0, 75, 5
+        trim_lefts = 0, 5
+        raw_lens = []
+        for trunc_len, trim_left in product(trunc_lens, trim_lefts):
+            if trunc_len == trim_left and trunc_len > 0:
+                with self.assertRaises(RuntimeError):
```

Comment:
Can you make this an `assertRaisesRegex` --- they are @jairideout's favorite and i think he would be excited to see one here:

```python
with self.assertRaisesRegex(RuntimeError, "No matches found"):
```